### PR TITLE
Fix `dataMigrate` command import error via babel register

### PR DESCRIPTION
**Pull request by dac09**
Will fill in details tomorrow. Putting here for DP to get early access :) 

> dp edit below

The `dataMigrate` commands were throwing a strange import error regarding `db.js` and Prisma Client. This PR resolves the import error.

Screen Shot 2021-02-22 at 11 58 33 PM

### CODE REVIEW

File: packages/cli/src/commands/dataMigrate/up.js [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
+import babelRequireHook from '@babel/register'
 import Listr from 'listr'
 import VerboseRenderer from 'listr-verbose-renderer'
 import terminalLink from 'terminal-link'
 
 import { getPaths } from 'src/lib'
 import c from 'src/lib/colors'
 
-require('@babel/register')
+babelRequireHook({
```
Comment (thedavidprice) [inline packages/cli/src/commands/dataMigrate/up.js:12]: seeing this only increases my dislike for babel

**Comment by thedavidprice** [comment]
Progress. Getting an error but it's a new one:
```bash
$ yarn rw dm install                      
yarn run v1.22.10
$ /Users/price/Repos/xx-delete/node_modules/.bin/redwood dm install
/Users/price/Repos/xx-delete/node_modules/@redwoodjs/cli/node_modules/yargs/build/index.cjs:2772
                throw err;
                ^

Error: @prisma/client did not initialize yet. Please run "prisma generate" and try to import it again.
In case this error is unexpected for you, please report it in https://github.com/prisma/prisma-client-js/issues/390.
    at new PrismaClient (/Users/price/Repos/xx-delete/node_modules/.prisma/client/index.js:3:11)
    at Object.<anonymous> (/Users/price/Repos/xx-delete/api/src/lib/db.js:6:19)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Module._compile (/Users/price/Repos/xx-delete/node_modules/pirates/lib/index.js:99:24)
    at Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Object.newLoader [as .js] (/Users/price/Repos/xx-delete/node_modules/pirates/lib/index.js:104:7)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by dac09** [comment]
> Progress. Getting an error but it's a new one
I think this is possibly as expected. We could get datamigrate to build the prisma client before, but then it would slow down the whole command. 

Off the top of my head I think the prisma client is built after rw upgrades and before building api automatically. Should we also have it here?

**Comment by thedavidprice** [comment]
@dac09 The new `rw prisma migrate dev` command _does_ generate the Prisma Client. So that's why this error is suspicious to me:
```bash
Error: @prisma/client did not initialize yet. Please run "prisma generate" and try to import it again.
```

I did try running `rw prisma generate` before I ran `rw dataMigrate install` and it still gave the same error. So I don't think it's actually the problem here.

**Pull request merged**